### DISCUSSION TITLE
How do I update my version of code-server?

### DISCUSSION THREAD

**Discussion by gaboc4** [coder/code-server#2087] [Q&A] [answered]
I am looking forward to the VS Code update that potentially fixed the iPad copy/paste issue and was wondering how I can update the version of code-server I currently have. I installed in on an Ubuntu server 20.04 machine using the install script. thanks!

**Comment by code-asher** [comment, marked as answer]
We don't have a release with that fix yet but we're working on getting it out. Once we do you should be able to re-run the install script and it'll install the new version.

**Reply by WhitCrow** [reply]
how to update by  myself. Because i use code-server in a off-line envior

**Reply by code-asher** [reply]
You'll need to do again whatever you did to install it in the first place. For example if you downloaded the GitHub release then you could download the new release and copy it over the old one.

**Reply by WhitCrow** [reply]
thank you very much
